woocommerce/navigation
# Check for dom root element before rendering

Fixes #47 

Fixes console warnings from lack of dom element on non-wc pages.

### Testing

1. Open your console.
1. Navigate to both wc and non-wc pages.
1. Make sure no console warnings exist related to this plugin.
